Training dataset size: 400
Training dataset size: 400
  0%|                                                  | 0/1000 [00:00<?, ?it/s]
Loss after 00003 examples: 1.009
Loss after 00007 examples: 0.966
Loss after 00011 examples: 0.942
Loss after 00015 examples: 0.921
Loss after 00019 examples: 0.909
  0%|                                                  | 0/1000 [00:56<?, ?it/s]
Traceback (most recent call last):
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 222, in model_pipeline
    train(model,train_loader,test_loader,criterion,eval_criterion,config)
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 120, in train
    cup_loss,disk_loss = test(model,test_loader,eval_criterion,config,best_valid_score,batch_ct)
  File "/home/kebl6872/REFUGE_4YP/Run/train_neat.py", line 86, in test
    data_str = f"Valid score improved from {best_valid_score[0]:2.8f} to {val_score:2.8f}. Saving checkpoint: {checkpoint_path_lowloss}"
NameError: name 'checkpoint_path_lowloss' is not defined
model tested on 400 imagesval_score: 0.14302324112784523 f1_scores [0.87296068 0.26459897 0.02144751 0.10865694]